### ISSUE TITLE
[TODO List] New Editor and UI

### ISSUE THREAD

**Issue by michaelsakharov** [michaelsakharov/Prowl#126]
The New UI system is nearing completion and is almost at feature parity with the previous Dear ImGUI editor!
These are the remaining tasks:

Bugs:
- [x] Transform Gizmo, Multi-select scale isn't working
- [x] Popup - Clicking on the parent node that created the popup doesn't close the popup
- [x] Input Fields hold focus too well
- [x] View Manipulator struggled with up and down due to Scene View camera Clamping
- [x] When Renaming Gameobject the name text moves down by a couple of pixels
- [x] Docking Separator can be selected when interactables/blockers exist above it
- [x] Numerical Input Fields need to support '-' for negative values
- [x] Can't drag from Asset Browser to the inspector, Selection is happening On click, not release
- [x] GUI Scaling breaking input
- [x] Inspector Editor only displays Null on component fields should still show full field
- [x] Scroll is broken
- [x] Tooltips cause Clip Rects to break?
- [x] Asset Browser is slow due to not using new Asset Database Cache

Missing Features:
- [x] Transform Gizmo, Uniform Scaling when ctrl is held
- [x] Asset Tree & Browser, Drag files/folders into folders
- [x] Asset Tree & Browser, Right Click Menu
- [x] Asset Tree & Browser, Rename
- [x] Component Right click menu
- [x] File Dialog
- [x] GUI Attributes
- [x] Layermask editor
- [x] List & Array support in Inspector
- [x] Inspector Add Component
- [x] Console window is missing its options
- [x] Support dragging material into scene view
- [x] Duplicate selected object from Scene View
- [x] Horizontal scroll

Once these are done the new Editor is finished and will be merged into the main branch!

**Comment by PaperPrototype** [comment]
YESSSSSS YESSSSSSSSSS YESSSSSSSSSSSS
